charts/stable/valheim/values.yaml

```diff
@@ -46,11 +46,11 @@ service:
     enabled: true
     type: LoadBalancer
     ports:
-      gameserver:
+      game:
```

Comment:
I don't disagree with this or `query`, but it seems arbitrarily different.  What's the purpose other than "putting your own stamp" on it?  I tried to find consistency in other Steam servers but there wasn't any that I found.

Reply:
Keep it simple for services... its a steam game and users can infer is a query for steam.